### DISCUSSION TITLE
Is this project still active?

### DISCUSSION THREAD

**Discussion by IAteYourBacon** [immense/Remotely#472] [Q&A]
https://remotely.one/ seems to be down?

**Comment by Basti-Fantasti** [comment]
It would be good to get an official statement of the new repo maintainer about the planned future of this project.

There have been no releases and no direction in which this project will move in the near future.

**Comment by ZieMa23** [comment]
As I'm using this tool frequently it would be great to know the future of this project. A statement from the dev team would be appreciated.

**Comment by stevesobol** [comment]
Hi! I'm a committer, but I'm not the project owner. I have reached out to the project owner for an official reply.

**Reply by Basti-Fantasti** [reply]
Thanks for your support. 
It would be good to know in which direction this project is currently moving.

Best regards
Bastian

**Reply by stevesobol** [reply]
> ImmyBot experienced explosive growth this year and we only have 4 developers which is why there hasn't been much activity here. We did a big push earlier this year to build an integration between Remotely and ImmyBot but ran into some technical challenges trying to keep only one agent on machines, but we don't want to cannibalize Remotely for the gain of ImmyBot. We want to keep Remotely fully operable as a standalone project. When we resume this initiate later this year, you will likely abstraction of the Remotely.Desktop code into a shared library that can be consumed by Remotely as well as other agents like ImmyBot.

https://github.com/immense/Remotely/blob/master/README.md

Darren just updated the README this morning. There's more that you can read if you click on the link to the readme.

There are longtime open issues that I have been interested in working on, as an individual, since long before ownership of the project changed hands, but I've not had the bandwidth to work on those issues, so I completely understand where he's coming from.

**Comment by bitbound** [comment]
Just wanted to chime in on this.  (I'm the original author of Remotely.)

I'm going to be helping out a little with maintaining Remotely until Darren has some extra time/employees/etc. to focus on the project.  Rest assured, he's still committed to the project.

**Reply by skutter-de** [reply]
Can you fix an issue I have? The Client on Windows 7 cannot seem to connect to the server. Is there any way I can debug this?

**Reply by bitbound** [reply]
I personally won't be doing any Windows 7 work.  Apologies.
